concreate/tools.py

```diff
@@ -100,7 +107,8 @@ def merge_lists(list1, list2):
             else:
                 for v1 in list1_dicts:
                     if v2['name'] == v1['name']:
-                        merge_dictionaries(v1, v2)
+                        if isinstance(v1, dict) and isinstance(v2, dict):
```

Comment:
Name here is to be sure if we have list of dictionaries, that we are merging proper ones - thiw will just pair them in order, Im not sure if this works correctly.